stratosphere/pm/source/pm_registration.cpp

```diff
@@ -13,9 +13,12 @@
  * You should have received a copy of the GNU General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
+
+#include <atomic>
```

Comment:
Is this actually used?

Reply:
Probably not! Noting to myself for PM rewrite.

This was added because the loader title tracking briefly lived in PM. Then I realized I couldn't have loader talk to pm, or deadlocks would occur, so I moved it into loader. Forgot to remove the header include then.